Adafruit_MQTT_Client.cpp

```diff
@@ -64,10 +67,6 @@ uint16_t Adafruit_MQTT_Client::readPacket(uint8_t *buffer, uint16_t maxlen,
       //DEBUG_PRINTLN((uint8_t)c, HEX);
       len++;
 
-      if (maxlen == 0) { // handle zero-length packets
```

Comment:
This was being called after the `client->read` and causing it to incorrectly read a byte from another packet - breaking parsing for the next packet(s) in the queue.

Reply:
Great, thanks for fixin'